tests/test_convert_lit_checkpoint.py

```diff
@@ -191,3 +191,30 @@ def test_maybe_unwrap_state_dict(tmp_path):
     with mock.patch("scripts.convert_lit_checkpoint.maybe_unwrap_state_dict") as maybe_unwrap:
         convert_lit_checkpoint(checkpoint_name=ckpt_name, out_dir=tmp_path, model_name=model_name)
     maybe_unwrap.assert_called()
+
+
+def test_check_conversion_supported_adapter(tmp_path):
```

Comment:
Tmp_path is unused across tests

```suggestion
def test_check_conversion_supported_adapter():
```

Reply:
updated in [9ab402f](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/9ab402f44120917cd6f0f4b049fcad1be62be24d)